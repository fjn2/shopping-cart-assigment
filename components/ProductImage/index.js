const ProductImage = ({ name, available }) => {
  return (
    <div style={{ aspectRatio: '1', filter: `grayscale(${available ? 0 : 1})` }}>
      <img variant="top" src={`https://avatars.dicebear.com/api/jdenticon/${name}.svg`} />
    </div>
  )
}

export default ProductImage