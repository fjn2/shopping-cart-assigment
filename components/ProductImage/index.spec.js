import { shallow } from 'enzyme';
import ProductImage from './';

describe('ProductImage', () => {
  it('Should render correctly', () => {
    const component = shallow(<ProductImage />)
    
    expect(component).toMatchSnapshot()
  })
})