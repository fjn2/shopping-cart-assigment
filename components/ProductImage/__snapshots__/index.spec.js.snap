// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage Should render correctly 1`] = `
<div
  style={
    Object {
      "aspectRatio": "1",
      "filter": "grayscale(1)",
    }
  }
>
  <img
    src="https://avatars.dicebear.com/api/jdenticon/undefined.svg"
    variant="top"
  />
</div>
`;
